e2e/utils/pages/homepage.page.ts

```diff
@@ -0,0 +1,68 @@
+import {
+	HERO_ANIMATION_CANVAS,
+	LOGIN_BUTTON,
+	TOKENS_SKELETONS_INITIALIZED
+} from '$lib/constants/test-ids.constant';
+import { type InternetIdentityPage } from '@dfinity/internet-identity-playwright';
+import { type Page } from '@playwright/test';
+import { HOMEPAGE_URL, LOCAL_REPLICA_URL } from '../constants/e2e.constants';
+
+type HomepageParams = {
+	page: Page;
+};
+
+type HomepageLoggedInParams = {
+	iiPage: InternetIdentityPage;
+} & HomepageParams;
+
+export class Homepage {
+	readonly page: Page;
```

Comment:
I guess those (I said those because same for `iiPage` in the other class) can be private? 

```
private readonly page: Page;

// or

readonly #page: Page;
```

Reply:
Yes, now I made them both private.